documentation/README.md

```diff
@@ -8,28 +8,33 @@ The testing framework, methodology, documentation and testing artifacts are stor
+| General Architecture and Requirements | [DASH general](./general/README.md)|        |
+| Data plane                            | [Data plane](./dataplane/README.md)         |
+| High-Availability (HA)                | [High-Availability](./high-avail/README.md) |
+| gNMI Northbound API                   | [gNMI Northbound API](./gnmi/README.md)     |
+| SAI Southbound API                    | [SAI Southbound API](./sai/README.md)       |
 
 ## Service Specifications and Requirements
+
 DASH devices may implement one or more of the following services.
+
 They shall conform to each service's design specifications and compliance requirements.
-| Topic   | Links to Folders |
-| ------- | ---------|
-| Load Balancer Service                | [Parent Folder](load-bal-service/README.md) \| [ Design](load-bal-service/design/README.md) \| [Compliance Requirements](load-bal-service/requirements/README.md)|
-| VNET-to-VNET Service                         | [Parent Folder](vnet2vnet-service/README.md) \| [ Design](vnet2vnet-service/design/README.md) \| [Compliance Requirements](vnet2vnet-service/requirements/README.md)|
-| Service Tunnel & Private Link Service                          | [Parent Folder](stpl-service/README.md) \| [ Design](stpl-service/design/README.md) \| [Compliance Requirements](stpl-service/requirements/README.md)|
-| VNET Peering Service                          | [Parent Folder](vnet-peering-service/README.md) \| [ Design](vnet-peering-service/design/README.md) \| [Compliance Requirements](vnet-peering-service/requirements/README.md)|
-| Express Route (ER) Service                    |  [Parent Folder](express-route-service/README.md) \| [ Design](express-route-service/design/README.md) \| [Compliance Requirements](express-route-service/requirements/README.md)|
-| Encryption Gateway Service                    |  [Parent Folder](encrypt-gw-service/README.md) \| [ Design](encrypt-gw-service/design/README.md) \| [Compliance Requirements](encrypt-gw-service/requirements/README.md)|
+
+| Topic                                 | Documents                                                         |
+| --------------------------------------| ------------------------------------------------------------------|
+| Load Balancer Service                 | [Load Balancer Service](./load-bal-service/README.md)             |
+| VNET-to-VNET Service                  | [VNET-to-VNET Service](./vnet2vnet-service/README.md)             |
+| Service Tunnel & Private Link Service | [Service Tunnel & Private Link Service](./stpl-service/README.md) |
+| VNET Peering Service                  | [VNET Peering Service ](./vnet-peering-service/README.md)         |
+| Express Route (ER) Service            | [Express Route (ER) Service](./express-route-service/README.md)   |
+| Encryption Gateway Service            | [Encryption Gateway Service](./encrypt-gw-service/README.md)      |
 
 
 # Organization of Design & Requirements Documents
```

Comment:
This section is no longer accurate, perhaps it can be deleted.

Reply:
@chrispsommers Good to hear from you. 
I can remove the entries in the table. What about the folders they point to, should we keep them around for now?

Reply:
@chrispsommers Going out for a while. I'll work on it when back.

Reply:
HI Michael, likewise nice to see you around. I was referring to the description of separate design and requirements folders, which this PR eliminates. I don't think the tables are an issue, it's the "separate folders" idea we're eliminating. So that paragraph is probably obsolete. As far as the "relationships and flow of documents" it can stand as-is for now unless there's some easy correction re: folder hierarchy. I need to give it a fresh pass or eliminate it, it might be too pedantic and not likely to be adhered to.

Reply:
Removed "Organization of Design & Requirements Documents" section because obsolete.